### DISCUSSION TITLE
[Feature] Duplicate/clone albums

### DISCUSSION THREAD

**Discussion by Godd67** [immich-app/immich#9360] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Lets say, I had a wonderful carribean vacation with my wife and created an album called "Carribean24" with tons of photos of tropical birds, plants, beaches and my wife topless. Now she wants to share this album with her parents except topless part :)
I'd love to just clone already existing album and remove unneeded pictures in a couple of click instead of recreate it .
What do you think?

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by NicholasFlamy** [comment]
Interesting idea, but if you need a solution for right now, you can Select All (select a picture then hit the square with dots around it which is the Select All button) the pictures (from within the Album) and then hit the Plus button and hit Add to Album.
